lib/form/js/ux.js

```diff
@@ -27,13 +27,12 @@
 
     // Run the script
     script = allPlugins[inp.parent.type].scripts[inp.script.function](input, settings);
-
   };
 
   // Add all event listeners
   Object.keys(allInputs).forEach(function (id) {
     if (allInputs[id].hasOwnProperty('script')) {
-      allIDs[id].addEventListener(allInputs[id].script.on, script);
+      script(allIDs[id]);
```

Comment:
@Snugug @scottnath 
calls script for every input.
In checkboxes, it might be redundant to call script for each checkbox.
[https://github.com/punchcard-cms/input-plugin-checkbox/blob/master/index.js](url)